internal/proto/controller/api/resources/accounts/v1/account.proto

```diff
@@ -82,90 +128,109 @@ message Account {
     ];
   }
 
-  // Output only. managed_group_ids indicates IDs of the managed groups that currently contain this account
-  repeated string managed_group_ids = 110 [json_name = "managed_group_ids"]; // @gotags: `class:"public" eventstream:"observation"`
+  // Indicates IDs of the managed groups that currently contain this account
+  repeated string managed_group_ids = 110 [
+    json_name = "managed_group_ids",
+    (google.api.field_behavior) = OUTPUT_ONLY
+  ]; // @gotags: `class:"public" eventstream:"observation"`
 
-  // Output only. The available actions on this resource for this user.
-  repeated string authorized_actions = 300 [json_name = "authorized_actions"]; // @gotags: `class:"public"`
+  // The available actions on this resource for this user.
+  repeated string authorized_actions = 300 [
+    json_name = "authorized_actions",
+    (google.api.field_behavior) = OUTPUT_ONLY
+  ]; // @gotags: `class:"public"`
 }
 
-// Attributes associated only with Accounts with type "password".
+// Attributes associated only with accounts with type "password".
 message PasswordAccountAttributes {
-  // The login name of this Account. This is unique per Auth Method.
+  // The login name of this account. This is unique per auth method.
   string login_name = 10 [
     json_name = "login_name",
     (custom_options.v1.generate_sdk_option) = true,
     (custom_options.v1.mask_mapping) = {
       this: "attributes.login_name"
       that: "LoginName"
-    }
+    },
+    (google.api.field_behavior) = REQUIRED
   ]; // @gotags: `class:"sensitive"`
 
-  // The password for this Account.
-  google.protobuf.StringValue password = 20 [(custom_options.v1.generate_sdk_option) = true]; // @gotags: `class:"secret"`
+  // The password for this account.
+  google.protobuf.StringValue password = 20 [
+    (custom_options.v1.generate_sdk_option) = true,
+    (google.api.field_behavior) = REQUIRED
```

Comment:
Same questions as for aliases, thanks for bringing it up.

Regarding `INPUT_ONLY`, I don't think it has any effect on the OpenAPIv2 translation, so while it may be appropriate here, it doesn't really matter.